### PR TITLE
[EASY] Add Balancer Pool Fees Spell to Gnosis Chain

### PR DESCRIPTION
**Pull request by thetroyharris**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [x] `coin_id` represents the ID of the coin on coinpaprika.com
* [x] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [x] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [x] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [x] where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by thetroyharris** [comment]
For anyone reviewing, this spell is the exact same spell that is already in production for other blockchains. Hopefully, that helps. Thank you!

File: models/balancer/gnosis/balancer_v2_gnosis_pools_fees.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                                    \'["metacrypto", "jacektrocinski", "thetroyharris"]\') }}'
+    ) 
+}}
+
+{% set event_signature = '0xa9ba3ffe0b6c366b81232caab38605a0699ad5398d6cce76f91ee809e322dafc' %}
+{% set project_start_date = '2022-11-02' %}
+
+WITH registered_pools AS (
+    SELECT DISTINCT
+        `poolAddress` AS pool_address
```
Comment (Hosuke) [inline models/balancer/gnosis/balancer_v2_gnosis_pools_fees.sql:21]: Maybe we can remove the `?
```suggestion
        poolAddress AS pool_address
```

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @thetroyharris 

[Sample balancer_v2_gnosis_pools_fees](https://dune.com/queries/2628709?d=1)

Also please take a look in the sample date to see if the range of `swap_fee_percentage` is expected.

**Comment by thetroyharris** [comment]
Merged a, now closed, 3 line PR #3521  regarding the aggregate `balancer_bpt_prices` Spell to this commit as it is another tiny change and felt it didn't need to be its own PR. Lmk if you all would prefer for me to have left it as it's own PR.

**Review by Hosuke — APPROVED** [review]
LGTM. for the new changes in `balancer_bpt_prices`.

**Pull request merged**